packages/zero-runtime/src/processors/styled.ts

```diff
@@ -192,8 +202,93 @@ export class StyledProcessor extends BaseProcessor {
     }`;
   }
 
+  private generateArtifacts() {
+    const artifacts: [Rules, Replacements][] = this.collectedStyles.map(([className, cssText]) => {
+      const rules: Rules = {
+        [`.${className}`]: {
+          className,
+          cssText,
+          displayName: this.displayName,
+          start: this.location?.start ?? null,
+        },
+      };
+      // @TODO - Refactor for finer location tracking in original code.
+      const replacements: Replacements = [
+        {
+          length: cssText.length,
+          original: {
+            start: {
+              column: this.location?.start.column ?? 0,
+              line: this.location?.start.line ?? 0,
+            },
+            end: {
+              column: this.location?.end.column ?? 0,
+              line: this.location?.end.line ?? 0,
+            },
+          },
+        },
+      ];
+      return [rules, replacements];
+    });
+    artifacts.forEach((artifact) => {
+      // Wyw-in-JS accesses artifacts array to get the final
+      // css definitions which are then exposed to the bundler.
+      this.artifacts.push(['css', artifact]);
+    });
+  }
+
+  private buildForTemplateTag(values: ValueCache): void {
+    const templateStrs: string[] = [];
+    // @ts-ignore @TODO - Fix this. No idea how to initialize a Tagged String array.
+    templateStrs.raw = [];
+    const templateExpressions: Primitive[] = [];
```

Comment:
Does this help? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw